common/test/services/prismic/opening-times.test.ts

```diff
@@ -93,7 +94,7 @@ describe('opening-times', () => {
     it('does not include a date more than once', () => {
       const result = getOverrideDatesForAllVenues(venues);
       const uniqueDates = new Set(
-        result.map(date => date.overrideDate?.toString())
+        result.map(date => date.overrideDate.toString())
```

Comment:
does this assume they all have override dates?

Reply:
Yup, this is enforced by the type system – the type here is `OverrideDate`, which always has the `overrideDate` field specified: https://github.com/wellcomecollection/wellcomecollection.org/blob/d79f36395c845bfdc6be8ab68bc2a2eabe5c42af/common/model/opening-hours.ts#L15-L18